Codecademy/gamut
# feat(PreviewTip): add preview tip

### Overview


Add new PreviewTip component, restructures some shared `tip` components, and adds a 3ms delay to to hover/focus tips (like PreviewTip + ToolTip)


### PR Checklist

- [x] Related to designs: [here](https://www.figma.com/design/ReGfRNillGABAj5SlITalN/%F0%9F%93%90-Gamut?node-id=41538-55277&node-type=frame&t=0TDLFJfuUyRaDgV3-0)
- [x] Related to JIRA ticket: [gm-645]

#### Testing Instructions

1. Go to the PreviewTip story 
2. Hover over one of the tips and see the text
3. Turn on VO 
4. Tab into the tip
5. Hear the tip announced
6. Do the same for the floating variant
7. ...
8. Profit!

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/38455)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| monorepo       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/7550)  | [Portal Env](http://www.google.fr/ 'Named link title')   |
